### PR TITLE
feat: ledger connected menu

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Adds a Ledger connected menu to connect additional chains

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

## Risk

None, isolated to Ledger under flag

## Testing

- Connect Ledger
- Ensure clicking "Connect accounts" opens the Ledger accounts/chains selection menu again

### Engineering

- ☝🏽 

### Operations

- ☝🏽 

## Screenshots (if applicable)

image
image

### CODE REVIEW

**Comment by gomesalexandre** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #5537** <a href="https://app.graphite.dev/github/pr/shapeshift/web/5537" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/shapeshift/web/5537?utm_source=stack-comment).

File: src/components/Layout/Header/NavBar/WalletConnectedMenu.tsx [hunk 1]
```diff
@@ -37,6 +37,8 @@ const ConnectedMenu = memo(
       [connectedType],
     )
 
+    console.log({ ConnectMenuComponent })
+
```
Comment (gomesalexandre) [inline src/components/Layout/Header/NavBar/WalletConnectedMenu.tsx:41]: revert
```suggestion
```

File: src/components/Layout/Header/NavBar/WalletConnectedMenu.tsx [hunk 2]
```diff
@@ -114,6 +116,7 @@ export const WalletConnectedMenu = ({
     )
   }, [])
 
+  console.log('here')
```
Comment (gomesalexandre) [inline src/components/Layout/Header/NavBar/WalletConnectedMenu.tsx:119]: revert
```suggestion
```

File: src/context/WalletProvider/Ledger/components/LedgerMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    })
+
+    dispatch({ type: WalletActions.SET_WALLET_MODAL, payload: true })
+  }, [dispatch])
+
+  return (
+    <>
+      <MenuDivider />
+      <MenuItem justifyContent='space-between' onClick={handleClick}>
+        {translate('Connect Accounts')}
```
Comment (gomesalexandre) [inline src/context/WalletProvider/Ledger/components/LedgerMenu.tsx:28]: No translation yet, agreed with @reallybeard  to author a follow-up PR with translations paths once all Ledger PRs are merged, which will act as a PR review of copies

**Comment by kaladinlight** [comment]
Only concern here is multi ledger connection scenario. We really want to make sure a user can't accidentally connect a ledger and get accounts on a couple apps, and then connect another ledger and connect the other apps and have a mixed set of accounts. The verify step would help catch this assuming it is mandatory, but the one shot connect scenario seems the safest (ie. going back into that modal resets everything, and you need to connect each one again). Not as clean of UX, but don't want mixed accounts with all of the cached/unchained only fetch logic. Thoughts?

**Comment by kaladinlight** [comment]
per discussion with @gomesalexandre this does not change the current behavior, so going to approve as is, but we should follow up as a group to just discuss and brainstorm the possible edge cases if we would like to address or change things in a follow up

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**